packages/e2e-tests/src/specs/editor/media/insertWebMVideo.js

```diff
@@ -70,23 +74,19 @@ describe('Inserting WebM Video', () => {
 
     const fileName = await uploadMedia('small-video.webm');
     uploadedFiles.push(fileName);
-
-    await page.waitForSelector(
-      `[data-testid="mediaElement-video"] [src*="${fileName}"`
-    );
+    const document = await getDocument(page);
+    await getByTestId(document, 'mediaElement-video');
```

Comment:
This would need to be in a `waitFor()`, otherwise this is not equivalent to `waitForSelector()`. Also the `[src*="${fileName}]` part is missing.

Reply:
Still exploring this library. This is a WIP and draft.